### PR TITLE
Add Native ad capability to AdYouLike adapter

### PR DESCRIPTION
**Pull request by guiann**
AdYouLike adapter now supports Native ads.

- test parameters for validating bids
```
    var adUnits = {
    "code": "test-div",
    "mediaTypes": {
      "native": {
        "image": {
          "required": true,
        },
        "title": {
          "required": true,
          "len": 80
        },
        "cta": {
          "required": false
        },
        "sponsoredBy": {
          "required": true
        },
        "clickUrl": {
          "required": true
        },
        "privacyIcon": {
          "required": false
        },
        "privacyLink": {
          "required": false
        },
        "body": {
          "required": true
        },
        "icon": {
          "required": true,
          "sizes": []
        }
      }
      bids: [{
        bidder: "adyoulike",
        params: {
          placement: 194 f787b85c829fb8822cdaf1ae64435,
          DC: "fra01", // Optional for set the data center name
        }
      }]
    };
```

- contact email of the adapter’s maintainer
guillaume.andouard@adyoulike.com

- [*] official adapter submission

Link to prebid.github.io update  PR:

https://github.com/prebid/prebid.github.io/pull/2633

### CODE REVIEW

**Comment by guiann** [comment]
Theunit test issue is from another bid adapter ...
1) bid response is valid
     waardexBidAdapter interpretResponse

**Comment by harpere** [comment]
ping @GLStephen

File: modules/adyoulikeBidAdapter.md [hunk 1]
```diff
@@ -11,19 +11,49 @@ Banner formats are supported.
...
+        },
+        "icon": {
+          "required": true,
+          "sizes": []
+        }
+      }
+      bids: [{
+        bidder: "adyoulike",
+        params: {
+          placement: 194 f787b85c829fb8822cdaf1ae64435,
```
Comment (Rothalack) [inline modules/adyoulikeBidAdapter.md:54]: Looks like an accidental space in the placement. Very minor, but it could throw some people off.

**Comment by Rothalack** [comment]
I wish I could find more substantive feedback, but it looks good!

File: modules/adyoulikeBidAdapter.md (hunk 1 above)
Comment (guiann) [reply modules/adyoulikeBidAdapter.md:54]: Indeed, will be fixed in a future release. thx

**Pull request merged**